lib/sle2docker/version.rb

```diff
@@ -1,3 +1,3 @@
 module Sle2Docker
-  VERSION = '0.4.2'.freeze
+  VERSION = '0.5.0'.freeze
```

Comment:
Why ? This should go into a separate PR, with an update on the changelog and all that. Please, remove this change.